WebRoot/css/login.css

```diff
@@ -134,6 +297,10 @@ FORM {
 	@cssShadow@:0px 0px 2px black;
 }
 
+.loginTitle{
+	padding: 0px 40px;
+	text-align: left;
```

Comment:
We can remove it as it inherited from #zLoginForm

Reply:
Done